webapp/public/js/domjudge.js

```diff
@@ -68,15 +68,17 @@ function sendNotification(title, options = {})
     } else {
         senttags = senttags.split(',');
     }
+
     if ( options.tag!==null && senttags.indexOf(options.tag)>=0 ) { return; }
 
     var link = null;
     if ( typeof options.link !== 'undefined' ) {
         link = options.link;
         delete options.link;
     }
-    options['icon'] = domjudge_base_url + '/apple-touch-icon.png';
 
+    options['icon'] = domjudge_base_url + '/apple-touch-icon.png';
+    
```

Comment:
Please be cautious introducing whitespace-only changes.

Reply:
I think I fixed this

Reply:
The newline seems to have moved, the rest is now ok